src/playground-code-editor.ts

```diff
@@ -960,6 +961,8 @@ export class PlaygroundCodeEditor extends LitElement {
         return 'google-html';
       case 'css':
         return 'css';
+      case 'jsx':
```

Comment:
Add `tsx` which should return `jsx` here (see other comment)

Reply:
sg done!